src/Styleguide/Components/Artwork/ImageCarousel.tsx

```diff
@@ -121,7 +129,7 @@ const ActionButtonsContainer = styled.div`
 `
 
 interface ImageCarouselProps {
-  src: string[]
+  images: Array<{ uri: string; aspectRatio: number }>
```

Comment:
I like this rename. Makes it much easier to read.

Reply:
Oh yeah absolutely, the previous one wasn’t really meant to be final in any way, neither is this one really :)